File: Model/Plant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WaterPlant.Model
{
    public class Plant
    {
        public int PlantId { get; set; }
        public string PlantName { get; set; }
        public string status { get; set; }
        public DateTime lastWateredAt { get; set; }
        public bool isWaterAllowed { get; set; }
    }
}

File: ClientApp/src/app/model/plant.ts
export class Plant {
    plantId?:number;
     plantName?:string;
     status?:string;
     lastWateredAt?:Date;
     isWaterAllowed?:boolean;
     canStopwatering=false;
      timeDifference;
      secondsToDday;
      minutesToDday;
      hoursToDday;
      daysToDday?:number;
      
}
